openwisp_radius/templates/html_password_reset_email.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'openwisp_utils/email_template.html' %}
+{% load i18n %}
+{% load l10n %}
+
+{% block mail_header %}
+{{ block.super }}
+<div class="subject">
+{% blocktrans %}Password reset on {{ site_name }}!{% endblocktrans %}
+</div>
+{% endblock %}
+
+{% block mail_body %}
+<div class="msg">
+    <p>
+        {% blocktrans %}
+        You're receiving this email because you requested a password reset for your user account at {{ site_name }}.
+        {% endblocktrans %}
+    </p>
+    <p>
+        {% trans "Please go to the following page and choose a new password:" %}
```

Comment:
Let's change this: Please click on the button below to open a page where you can choose a new password.